src/main/cpp/pop/Model.h

```diff
@@ -0,0 +1,113 @@
+#ifndef POPULATION_MODEL_H_INCLUDED
+#define POPULATION_MODEL_H_INCLUDED
+
+/**
+ * @file
+ * Header file for the population model class
+ */
+
+#include <memory>
+#include <vector>
+#include <boost/property_tree/ptree.hpp>
+#include "util/InclusiveRange.h"
+
+namespace stride {
+namespace population {
+
+struct TownPivot final
```

Comment:
I'm curious as to what a `TownPivot` is. Does it represent a pivot item from a list of towns? I don't think it does, given that there's a `std::vector<TownPivot>` field out there.

You might want to try documenting this `struct`. (If it's not too much work, that is.)